public/stylesheets/site/2.0/22-system-messages.css

```diff
@@ -58,13 +59,18 @@ form .notice, form ul.notes {
     box-shadow: 0px 0 3px #fff;
 }
 
-ul.notes, .error ul, .alert.flash ul {
+.notice.formatted p {
```

Comment:
Are you sure? There are a couple of pre-existing places that set `flash[:notice]` equal to an array, which produces a `ul` inside of an `li`, including:
- the admin skins page
- the co-creator requests page
- the troubleshooting page

This will change the look of those flash messages.

Reply:
Yeah, it's fine. We can give those special treatment for cases where there's only one message to display if we decide we're really bothered by single-item lists, but otherwise actually showing the list formatting makes sense.

I did see two that have no business being an array and should definitely be fixed at some point, but I'm fine with leaving them alone for now.

https://github.com/otwcode/otwarchive/blob/58f92a68bed5a28d9296db631b200be9d51facc3/app/controllers/skins_controller.rb#L117-L122

👆 Which also needs tweaks to the text.

https://github.com/otwcode/otwarchive/blob/58f92a68bed5a28d9296db631b200be9d51facc3/app/controllers/works_controller.rb#L396-L399

👆 Which needs tweaks to become i18n-ready. (I was having a hard time figuring out how to trigger this one; if it's one we see often, we can fix it in this PR rather than live with the oddness.)

Reply:
Okay, I've changed the CSS classes.

FYI, that second one actually isn't an array; it's using `<<` for string concatenation. So its formatting wouldn't be affected by the change.